TomeOfKnowledge.playground/Pages/Exploring Expressions.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,50 @@
+//: ## Exploring Expressions
+//: 
+//: Our expressions are more robust than what is needed for typical tabletop play. In fact, our generic expression system is capable of modeling expressions that would otherwise be considered too cumbersome or complex to physically roll. In order to help developers understand what their expressions actually represent, we have added various tools to make working with expressions as easy as possible.
+
+import DiceKit
+
+//: Lets start with the negative-sided die.
+
+let negativeDie = d(-12)
+
+//: A die with negative sides has results in the range `-sides...-1`. We have two ways to represent an expression as a string.
+
+let normalString = String(negativeDie)
+let verboseString = String(reflecting: negativeDie)
+
+//: `normalString` is the string returned from `negativeDie.description`. Our `description`s are intended to be readable and closely match regular dice notation. This also happens to be the text that playgrounds display to the right.
+//: `verboseString` is what is returned from `negativeDie.debugDescription`. These strings tend to do less to make the result readable, and are intended to show exactly what types are used. 
+
+//: When we look at the `description` of a die roll, we seperate the rolled value from the total number of sides with a |. So a d6 that rolled a 3 would be written as 3|6. Or with our special negative die: 
+
+let negativeResult = negativeDie.roll()
+
+//: Multiplication expressions are particularly interesting to look at. We will make one multiplication expression and see what happens when we make a third multiplication expression from the first one and a die.
+let oneD2 = 1 * d(2)
+let d3 = d(3)
+
+//: `oneD2` is of the type `MultiplicationExpression<Constant, Die>` because it is made by multiplying a `Consant` (the expression type `Int` is converted to) by a `Die`.
+
+oneD2 is MultiplicationExpression<Constant, Die>
+
+//: We can make a new multiplication expression with these two expressions. 
+
+let oneD2d3 = oneD2 * d3
+
+//: This expression works like a "normal" multiplication expression, but the multiplier is one half the time, and two the other half of the time. When we `evaluate()` a multiplication expression, we can see what each side of the expression evaluated to. First, an easy case:
```

Comment:
" but the multiplier is one half the time, and two the other half"

Let's say `1` and `2` instead, so they read easier. I had to re-read this a couple times to understand "one half" wasn't "1/2".